src/Controller/Component/CsrfComponent.php

```diff
@@ -56,7 +56,7 @@ class CsrfComponent extends Component
         'cookieName' => 'csrfToken',
         'expiry' => 0,
         'secure' => false,
-        'httpOnly' => false,
+        'httpOnly' => true,
```

Comment:
Let's imagine that the attacker can get the value of the cookie... or more simply done, the value of the hidden input.. How would he be able to use it? They would require the session cookie as well in order to actually be logged in on the site and make changes.